kinto_portier/authentication.py

```diff
@@ -66,7 +66,7 @@ def _verify_token(self, user_token, request):
         if encrypted_email is None:
             return None
 
-        email = decrypt(encrypted_email, user_token)
+        email = decrypt(encrypted_email.encode('utf-8'), user_token)
```

Comment:
Since `decrypt` and `encrypt` are implement locally in `crypto.py`, wouldn't you want to cast there?